test/e2e/framework/helpers.go

```diff
@@ -168,6 +170,23 @@ func (f *Framework) MustGetPods(t *testing.T, namespace string) *v1.PodList {
 	return pods
 }
 
+func (f *Framework) IsFeatureGateEnabled(name string) (bool, error) {
+	fg, err := f.OpenShiftConfigClient.ConfigV1().FeatureGates().Get(context.Background(), "cluster", metav1.GetOptions{})
+	if err != nil {
+		return false, err
+	}
+
+	enabledFeatureGates := fg.Status.FeatureGates[0].Enabled
+
+	for _, g := range enabledFeatureGates {
+		if g.Name == configv1.FeatureGateName(name) {
+			return true, nil
+		}
+	}
+
+	return false, nil
```

Comment:
Two suggestions here:

1. You could pass `t *testing.T` and add a line `t.Helper()` and then call `t.Fatal[f]` on an error. This is in line with the other helpers (see `MustGetPods` for example) and allows you to drop the additional layer in the test file.
2. Do we need or want to poll on `.FeatureGates().Get()`? We do this for other resources and I tihnk this might the test a bit more resilient against temporary cluster issues.

Reply:
I didn't use poll here since featuregate 'cluster' would be already existing on cluster after its installed. Do we want to poll in that case?

Reply:
It could be that there is an intermittent network issue that prevents the call from succeeding the first time. But we can go ahead with this and add a poll later if that turns out to be needed.